js/index.js

```diff
@@ -1,3 +1,169 @@
 function toggleMobileMenu(menu) {
     menu.classList.toggle('open');
+}
```

Comment:
Delete comented code

Reply:
Thank you for your reviews, Sir,
I've already made changes regarding the reviews given in this file. 
Yours.